src/components/receipt-view/data-section/DataSection.js

```diff
@@ -9,51 +9,103 @@ import ImpactDonutAggregate from './ImpactDonutAggregate';
 // Styles
 import styles from '../styles/data-section.module.css';
 
-const DataSection = () => (
+const LAND = 'land';
+const WATER = 'water';
+const EUTRO = 'eutro';
+const GHG = 'ghg';
+
+const colors = {
+  [LAND]: 'blue',
+  [WATER]: 'gold',
+  [EUTRO]: 'salmon',
+  [GHG]: 'green',
+};
+
+const minRatio = { title: 0.2, body: 0.3 };
+
+const DataSection = ({ cartState }) => {
+  const [windowDimensions, setWindowDimensions] = useState({
+    width: 0,
+    height: 0,
+  });
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowDimensions({
+        height: window.innerHeight,
+        width: window.innerWidth,
+        min: Math.min(window.innerWidth, window.innerHeight),
+      });
+    };
+
+    handleResize();
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
```

Comment:
So I've encountered this problem. For some reason, when building on Netlify, the window object is undefined and it breaks. So you have to wrap every instance of window with 

if (window !== undefined) { }

Reply:
Sorry, actually use 

if (typeof window !== `undefined`) {}

Reply:
I think it's `typeof window !== "undefined"` (notice that undefined is a string)